test_runner/src/main/kotlin/ftl/domain/README.md

```diff
@@ -0,0 +1,56 @@
+# Domain
+
+Owns business logic of the application.
+
+## Design
+
+The domain layer is exposing own API as a collection of public suspendable extension functions,
+which could be called a `top-level functions`.
+Each `top-level function` have own `execution context`, 
+can produce `structured output` during the `execution`
+and can be composed of one or many low-level functions.
+This layer could be considered as a standalone library, 
+that is providing access to business logic through pure kotlin functions. 
+
+
+### Top-level function
+
+Is public and accessible from the root of the domain package.
+IS responsible to run domain logic directly or compose it using low-level domain functions, utils, or data interfaces.
```

Comment:
Is the capitalization used to add emphasis or is it a mistake?
If mistake
```suggestion
Is responsible to run domain logic directly or compose it using low-level domain functions, utils, or data interfaces.
```

Reply:
Mistake, of course, thanks ;)